### DISCUSSION TITLE
旁路由模式下，手机浏览器无法访问google.com，电脑(windows+linux)正常，原因+解决方法。

### DISCUSSION THREAD

**Discussion by sunshe35** [vernesong/OpenClash#3050] [General]
**当前环境**：手机与电脑都正确配置了网关和DNS服务器为旁路由ip：192.168.10.159.

**产生问题**：手机无法访问google.com，但是可以访问google.com.hk以及youtube.com。

**根本原因**：路由器开启了ipv6，手机自动获取ipv6和ipv4地址，ipv6干扰了v4的网络。电脑浏览器linux和windows都能很好的处理这个问题，手机浏览器太辣鸡，处理不了。

解决方法：关闭路由器的ipv6即可。但是我的ipv6是一定要用的，不想关闭，所以推荐下面的做法。那就是在**手机连接wifi的界面配置代理**，参考如下图：。
![32917c166f18d6f91233cc020b12f0a](https://user-images.githubusercontent.com/16757906/221888813-c7e39966-6c9a-4aa7-860a-7142ec6f8aa6.jpg)

这只能初步解决问题，后面也可能遇到其他问题，到时候再说吧。**如果还有问题，用v2rayNG配置一个全局代理192.168.10.159:1050应该能解决问题**。

关于fake-ip模式自己有些困惑，好像只有linux的nslookup返回是正常的。但是win和linux又都可以访问goole.com，很困惑。

- linux下ipv4+ipv6
```
root@debianGUI2:/# nslookup google.com
Server:         192.168.10.159
Address:        192.168.10.159#53

Name:   google.com
Address: 198.18.0.4
```

- windows10 下 ，ipv4和ipv6返回的都不正常

```
# ipv4+ipv6
C:\Users\sunshe35>nslookup google.com
服务器:  cmcc.wifi
Address:  fd02:1:2:3:7242:d3ff:fe4f:19a9

非权威应答:
名称:    google.com
Address:  142.251.42.238

# ipv4
C:\Users\sunshe35>nslookup google.com
服务器:  UnKnown
Address:  192.168.10.159

非权威应答:
名称:    google.com
Address:  142.251.42.238
```

- Android ，基于termux。
```
$ nslookup google.com
Server:         8.8.8.8
Address:        8.8.8.8#53

Non-authoritative answer:
Name:   google.com
Address: 142.251.42.238
```

**Comment by taohuadongle** [comment]
折腾了我2个小时，感谢你的指引
